Makefile

```diff
@@ -16,7 +16,6 @@ build-windows:
 
 # build docker image
 build-docker:
-	GOOS=linux GOARCH=amd64 go build -ldflags ${LDFLAGS} -o bin/${BINARY_NAME}
```

Comment:
`build-docker` is not used. Why do you modify it?

Reply:
to create the docker image with `make build-docker` and not only with `docker build -t helm-wrapper .`

Reply:
So the command `make build-docker` did not work as expected before? If you fixed this, why did you change the readme?

Reply:
For me it is simpler run the docker build command with the  the tag and can change this direct to upload to a registry, other I must make Variable/Parameter or rename after build.